pkg/policy/l4.go

```diff
@@ -25,9 +25,16 @@ import (
 )
 
 type AuxRule struct {
-	Expr string `json:"expr"`
+	Expr     string `json:"expr"`
+	L7Parser string `json:"l7Parser"`
 }
 
+// HTTPStr for proxy type http
+const HTTPStr = "http"
```

Comment:
Let's make it clear that this is a parser type:
```
// L7ParserType is the type used to indicate what L7 parser to use and defines all supported types of L7 parsers
type L7ParserType string

const (
    // ParserTypeHTTP specifies a HTTP parser type
    ParserTypeHTTP L7ParserType = "http"

    [...]
)
```

Reply:
will do.. but looks like it needs typecasting to string everywhere
l4.L7Parser = string(ParserTypeKafka)

Reply:
done